docs/website/plugins/nikita-docs/gatsby-node.js

```diff
@@ -0,0 +1,100 @@
+
+const crypto = require("crypto")
+const path = require("path")
+const grayMatter = require("gray-matter")
+const { createFilePath } = require('gatsby-source-filesystem')
+
+exports.createSchemaCustomization = ({ actions }) => {
+  const { createTypes } = actions
+  createTypes(`
+    """
+    NikitaDocs Node
```

Comment:
docs is not the most appropriate name, I don't have a perfect one but page is slightly better